understanding/20/info-and-relationships.html

```diff
@@ -7,7 +7,16 @@
 </head>
 <body>
    <h1>Understanding Info and Relationships</h1>
-   
+
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Information about content structure is available to more people</dd>
+		<dt>Author task</dt><dd>Use code to reinforce relationships and information conveyed through presentation</dd>
+		<dt>Key beneficiaries</dt><dd>Users with reduced sight, hearing, or cognition</dd>
```

Comment:
@gundulaniemann said
> As the information should be programmatically determined, I suggest to add blind users to the list of beneficiaries.

"Users with reduced sight" is intended to include users who are blind, just as "Users with reduced hearing" is intended to include users who are deaf. As per other Beneficiaries comments, recommend not spending much time on this, as it will likely be removed.

Reply:
I fee then some streamlining might be useful, as sometimes blind users are mentioned, while for example sighted users using a screen reader also benefit.
I see it pays off only after clarification what will happen to the beneficiaries section.